src/php/Lang/Struct.php

```diff
@@ -19,9 +18,9 @@ abstract class Struct extends Table
 
     protected function validateOffset($offset)
     {
-        if (!in_array($offset, $this->getAllowedKeys())) {
+        if (!in_array($offset, $this->getAllowedKeys(), false)) {
             $keyName = Printer::nonReadable()->print($offset);
-            $structName = get_class($this);
+            $structName = self::class;
```

Comment:
Shouldn't this be `static` (Late static binding)?

Reply:
Yes, you are right -> https://stackoverflow.com/a/4719222/4988034

Reply:
Done. :)